packages/kyt-core/src/config/webpackHotDevClient.js

```diff
@@ -57,16 +57,31 @@ if (module.hot && typeof module.hot.dispose === 'function') {
   });
 }
 
-// Connect to WebpackDevServer via a socket.
-const connection = new SockJS(
-  url.format({
-    protocol: window.location.protocol,
-    hostname: window.location.hostname,
-    port: sockJSPort,
-    // Hardcoded in WebpackDevServer
-    pathname: '/sockjs-node',
-  })
-);
+let connection;
+if (!window.location.protocol.includes('http')) {
```

Comment:
Can this protocol be `https`?

Reply:
If you're `https` then the `.includes` will also match with `https`

Reply:
const protocol = 'https:'
protocol.includes('http') --> true